fcrepo-auth-webac/src/main/java/org/fcrepo/auth/webac/WebACFilter.java

```diff
@@ -154,21 +155,40 @@ private String findNearestParent(final String childPath) {
     private boolean isAuthorized(final Subject currentUser, final HttpServletRequest httpRequest) throws IOException {
         final String requestURL = httpRequest.getRequestURL().toString();
         final String repoPath = httpRequest.getPathInfo();
-
+        final boolean isAcl = requestURL.endsWith(FCR_ACL);
         final URI requestURI = URI.create(requestURL);
         log.debug("Request URI is {}", requestURI);
 
         // WebAC permissions
         final WebACPermission toRead = new WebACPermission(WEBAC_MODE_READ, requestURI);
         final WebACPermission toWrite = new WebACPermission(WEBAC_MODE_WRITE, requestURI);
         final WebACPermission toAppend = new WebACPermission(WEBAC_MODE_APPEND, requestURI);
+        final WebACPermission toControl = new WebACPermission(WEBAC_MODE_CONTROL, requestURI);
 
         switch (httpRequest.getMethod()) {
         case "HEAD":
         case "GET":
-            return currentUser.isPermitted(toRead);
+            if (isAcl) {
+                if (currentUser.isPermitted(toControl)) {
+                    log.debug("PUT allowed by {} permission", toControl);
+                    return true;
+                } else {
+                    log.debug("PUT prohibited by {} permission", toControl);
```

Comment:
Again, this log message is inaccurate (HTTP method is not `PUT` in this case).

Reply:
I will fix